src/web/styles/buttons.scss

```diff
@@ -13,8 +13,9 @@ button {
 }
 
 .primary-button:disabled {
-  background-color: var(--theme-background-sidepanel);
-  border-color: var(--theme-background-sidepanel);
+  background-color: var(--theme-table-border);
```

Comment:
Someone working on a theme would probably be surprised to find that when they change the table border color, it also changes the disabled button color.
I think a `theme-button-disabled` variable would make more sense, even though they're the same color value.